geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/Conversions.scala

```diff
@@ -180,6 +183,22 @@ object RichAttributeDescriptors {
   }
 }
 
+object RichSimpleFeatureType {
+
+  import RichAttributeDescriptors.RichAttributeDescriptor
+  import SimpleFeatureTypes.DEFAULT_DATE_FIELD
+
+  import scala.collection.JavaConversions._
+
+  implicit class RichSimpleFeatureType(val sft: SimpleFeatureType) extends AnyVal {
```

Comment:
merge with caution: in master there already is an "implicit class RichSimpleFeatureType" as a nested class of "object Conversions"